esmvaltool/recipes/recipe_diurnal_temperature_index_wp7.yml

```diff
@@ -0,0 +1,60 @@
+###############################################################################
+# namelist_diurnal_temperature_index.yml
+#
+# Description
+# Namelist for computing the  diurnal temperature indicator.
+#
+# Authors
+# Alasdair Hunter (BSC, Spain)
+# Nicolau Manubens (BSC, Spain)
+# Louis-Philippe  (BSC, Spain)
+#
+# Project
+# C3S Magic
+#
+#Reference
+#http://secif.ipsl.fr/images/SECIF/documents/Communication/fiche_invulnerable/RC_indicateur_EN.pdf
+#
+# This namelist is part of the ESMValTool
+###############################################################################
+---
+
+datasets:
+  #- {dataset: bcc-csm1-1,  type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1960,  end_year: 1990}
+  #- {dataset: bcc-csm1-1,  type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 1991,  end_year: 1995}
+  #- {dataset: bcc-csm1-1,  type: exp, project: CMIP5,  mip: Amon,  exp: historical,  ensemble: r1i1p1,  start_year: 2000,  end_year: 2004}
+  - {dataset: IPSL-CM5A-LR, type: exp, project: CMIP5,  mip: day,  exp: historical,  ensemble: r1i1p1,  start_year: 1961,  end_year: 1990}
+  - {dataset: IPSL-CM5A-LR, type: exp,  project: CMIP5,  mip: day,  exp: rcp85,  ensemble: r1i1p1,  start_year: 2020,  end_year: 2040}
+ # - {dataset: IPSL-CM5A-LR, type: exp,  project: CMIP5,  mip: day,  exp: rcp26,  ensemble: r1i1p1,  start_year: 2020,  end_year: 2040}
+
+latlon_roi: (/latmin,latmax,lonmin,lonmax/)
```

Comment:
This line should probably be removed.

Reply:
Following your suggestion, I've removed this line from all recipes. Thanks, @bouweandela !